src/API/Rest/Enums.cs

```diff
@@ -76,6 +76,11 @@ public enum InputInterfaceType
     /// </summary>
     public enum InferenceRequestType
     {
+        /// <summary>
+        /// Unknown request type
+        /// </summary>
+        Unknown,
+
```

Comment:
I'd add the FHIRResource here too, even though the operators do not support it, it is coming (just I don't know how exactly yet)

Reply:
Will add it when we do support it.